Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4243,9 +4244,13 @@ - (UIColor *)_cascadeInteractionTintColor
     return caretColorFromStyle.autorelease() ?: tintColorFromNativeView;
 }
 
-- (void)_updateInteractionTintColor:(UITextInputTraits *)traits
+- (void)_updateTextInputTraitsForInteractionTintColor
 {
-    [traits _setColorsToMatchTintColor:[self _cascadeInteractionTintColor]];
+#if !PLATFORM(WATCHOS)
+    auto tintColor = [self _cascadeInteractionTintColor];
```

Comment:
I'm not sure why this isn't on WatchOS

Reply:
We use `PUICTextInputContext` on watchOS, so this logic is (and remains) a no-op on watchOS.